lib/lettering/font.py

```diff
@@ -44,7 +45,11 @@ def getter(self):
             # This may be a font packaged with Ink/Stitch, in which case the
             # text will have been sent to CrowdIn for community translation.
             # Try to fetch the translated version.
-            return _(self.metadata.get(name))
+            original_metadata = self.metadata.get(name)
+            localized_metadata = ""
+            if original_metadata != "":
+                localized_metadata = _(self.metadata.get(name))
+            return localized_metadata
```

Comment:
@kaalleen I don't think this change does what you might have been thinking, based on the description on your commit.  The crowdin files are actually populated from the `bin/inkstitch-fonts-gettext` script.  I think (and hope) that babel ignores the `_(self.metadata.get(name))` call because it's not a simple string.

I included font names and descriptions in the translation strings in crowdin on purpose so they would get translated.  Do you think we should change that?

Reply:
No, I think it's a good idea to use crowdin for that. It just returns a lot of stuff if we send an empty string - I don't know for what reason. It works the way I indended it, I have been trying. I don't think using the old or new inkscape version would make a huge difference here. But we wouldn't need to call self.metadata again, just use original_metadata.

Reply:
Oh, I see I didn't read this correctly, sorry about that!  I understand how it works now.  Looks good to me.